src/components/unique-token/UniqueTokenAttributes.tsx

```diff
@@ -47,18 +20,31 @@ const UniqueTokenAttributes = ({
   slug,
   traits,
 }: UniqueTokenAttributesProps) => {
-  const sortedTraits = (sortList(
-    traits,
-    'trait_type',
-    'asc'
-  ) as typeof traits).map(trait => ({
-    ...trait,
-    color,
-    disableMenu,
-    slug,
-  }));
+  const sortedTraits = useMemo(
+    () =>
+      (sortList(traits, 'trait_type', 'asc') as typeof traits)
+        .filter(trait => trait.value && trait.trait_type)
```

Comment:
I added this filtering, because we were rendering trait without value as `null` it caused to render a blank space of 10px causing other traits to be offset from the screen edge

Reply:
And no it is not possible to render a trait that has an empty value, it was not possible to do this before as well

Reply:
ok